_data/retail.yml

```diff
@@ -423,4 +432,3 @@ websites:
       tfa: No
       twitter: zooplusDE
       facebook: zooplusde
-
```

Comment:
we do want trailing white space 😉 see https://github.com/2factorauth/twofactorauth/blob/master/.editorconfig#L10

Reply:
Looks like there's already one, there were was an extra one.